x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/rule_tag_badge.tsx

```diff
@@ -16,55 +16,61 @@ const tagTitle = i18n.translate(
   }
 );
 
+export type RuleTagBadgeOptions = 'all' | 'default';
+
+interface RuleTagBadgeBasicOptions {
+  isOpen: boolean;
+  onClick: React.MouseEventHandler<HTMLButtonElement>;
+  onClose: () => void;
+}
+
 export interface RuleTagBadgeCommonProps {
+  showAllTags?: boolean;
```

Comment:
nit: I understand it as if I set `showAllTags` to `false`, some tags will be hidden for an unknown reason. So maybe something like `expandTags`, `spreadTags`, `unfoldTags` would be a better choice.

Reply:
So maybe, we can do the other way and create `tagInPopover` as a boolean, I think that might be simpler

Reply:
Perfect, I liked it 👍🏻 !